trl/trainer/cpo_config.py

```diff
@@ -66,7 +66,7 @@ class CPOConfig(TrainingArguments):
 
     beta: float = 0.1
     label_smoothing: float = 0
-    loss_type: Literal["sigmoid", "hinge", "ipo", "kto_pair", "simpo"] = "sigmoid"
+    loss_type: Literal["sigmoid", "hinge", "ipo", "simpo"] = "sigmoid"
```

Comment:
Same here, if you can add an error here and educate users on what to do to replace `kto_pair` which is not supported anymore would be perfect 🙏